src/current/cockroachcloud/plan-your-cluster-basic.md

```diff
@@ -29,9 +29,9 @@ Refer to [Pricing](https://cockroachlabs.com/pricing) to see cost estimates of c
 
 ## Free vs. paid usage
```

Comment:
```suggestion
## Free and paid usage
```
Might require some link updates. Better for SEO.

Reply:
This section will be deleted before the release